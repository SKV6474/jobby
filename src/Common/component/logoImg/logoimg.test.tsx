import LogoImg from ".";
import renderer from "react-test-renderer";

describe("Logo Img", () => {
  test("Logo Img SnapShot", () => {
    const tree = renderer.create(<LogoImg />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
