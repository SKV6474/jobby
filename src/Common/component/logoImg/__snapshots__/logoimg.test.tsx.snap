// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Img Logo Img SnapShot 1`] = `
<div
  className="sc-beqWaB ekvAZh"
>
  <img
    alt="logo"
    className="sc-gueYoa jORccO"
    src="https://assets.ccbp.in/frontend/react-js/logo-img.png"
  />
</div>
`;
